combat-trainer.lic

```diff
@@ -104,8 +106,28 @@ class SetupProcess
 
     game_state.wield_whirlwind_offhand
   end
-
+  
+  def toggle_ah(mode)
+    @armor_hysteresis = mode
+    echo("*** armor hysteresis: #{@armor_hysteresis} ***")
+  end
+  
+  def toggle_armor_hysteresis?
```

Comment:
❌ The `?` at the end of the function name means this is a predicate. This function does not explicitly return a boolean and its return value is never used. Please remove the `?`